src/pages/hegel/guides/presuppositionless-thinking.md

```diff
@@ -1,8 +1,10 @@
+Presuppositionless thinking is a form of thinking that thinks without presupposing the outcome. Simply put, it is a commitment to a philosophical inquiry that does not proceed according to preconceived notions of how things are but is instead open to the possibility that things may be otherwise. At its most honest, the tenet of presuppositionlessness is the acceptance that our knowledge of things may be flawed or incomplete because the approach that philosophy has taken to comprehend a concepts may have itself been flawed. Thus, it is a voyage of exploration into the regions of reason that may or may not disclose something new. It is entirely possible that a presuppositionless approach may yield nothing at all, or nothing at all new. It is equally possible, however, that presuppositionlessness may show us something new about thought. We simply cannot presuppose prior to the ordeal.
 
-One may baulk at the idea that our knowledge of things may be flawed. What exactly is it that were claiming to be flawed? Doesn’t there need to be an argument for why we should take our knowledge to be flawed? Absolutely. One argument is this. Hegel takes seriously Kant’s discoveries in the transcendental dialectic – the self-movement of Reason is not merely an illusion but is in fact something fundamental about what it is to be rational. Yet, if we reflect on our typical understanding of concepts we will immediately notice that they bear no resemblance whatsoever to how reason appears to be. Reason is dialectical, it transforms into its opposite, and this movement of concepts appears to be essential to what reason is. For Kant, this is just an illusion precisely because this is not how the world appears to be. For Hegel, on the other hand, this should prompt us to wonder whether we have properly understood the nature of Reason and, thereby, the nature of thought itself.
+One may baulk at the idea that our knowledge of things may be flawed. What exactly is it that we're claiming to be flawed? Doesn’t there need to be an argument for why we should take our knowledge to be flawed? Absolutely. One argument is this. Hegel takes seriously Kant’s discoveries in the transcendental dialectic – the self-movement of Reason is not merely an illusion but is in fact something fundamental about what it is to be rational. Yet, if we reflect on our typical understanding of concepts we will immediately notice that they bear no resemblance whatsoever to how reason appears to be. In this regard reason is dialectical, namely, it transforms into its opposite, and this movement of concepts appears to be essential to what reason is. For Kant, this is just an illusion precisely because this is not how the world appears to be. For Hegel, on the other hand, this should prompt us to wonder whether we have properly understood the nature of Reason and, thereby, the nature of thought itself.
```

Comment:
@ridetoruin in this paragraph you've used reason with a capital R. What's the signifance of this? Do we want to capitalize terms? Why and why not?

Reply:
This is something I do in my own work. I find that it's makes comprehension smoother if I capitalise proper nouns like Reason or the Understanding. Do you think it's confusing?